src/js/wrapper.ts

```diff
@@ -144,7 +144,7 @@ export const NATIVE: SentryNativeWrapper = {
       };
 
       // Serialize and remove any instances that will crash the native bridge such as Spans
-      const serializedPayload = JSON.parse(JSON.stringify(payload));
+      const serializedPayload = normalize(payload);
```

Comment:
```suggestion
      const serializedPayload = JSON.parse(JSON.stringify(payload, walk));
```

@Krisztiaan I believe this would fix the broken tests, we can fix the normalization in another PR.

Reply:
Oh `normalize` actually calls `JSON.parse(JSON.stringify(...))` so not really.